### ISSUE TITLE
0 gets removed when encrypting a number string with leading 0

### ISSUE THREAD

**Issue by bitsofalex** [danang-id/simple-crypto-js#29]
When encrypting and decrypting a string with a leading 0, it removes the 0.

```
import SimpleCrypto from "simple-crypto-js"

let simpleCrypto = new SimpleCrypto("hYLiR5U1g2ppct1Aw8hf");
let encrypted = simpleCrypto.encryptObject("0419383728");
console.log(encrypted);

let decrypted = simpleCrypto.decrypt(encrypted);
console.log(decrypted + typeof decrypted);
```
output
```82605e28ea5cabda64bf8e631ab4ca79c8b81c7050f3b4ac635ece4230a21b8dnrqv90eqKqhgHAA6Al46Tw==2507d392ff251429cc48064e94523d830903b4905448d75c7e61335b031cd79e
419383728number

**Comment by danang-id** [comment]
Please provide `SimpleCrypto` version that you use. However, this may be a duplicate of #19 and has been fixed since version `2.4.1`.

**Comment by danang-id** [comment]
@bitsofalex I will closing this issue soon if there's no update on the issue.
